### PR TITLE
Misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell`

### CODE REVIEW

File: tests/data/test1108 [hunk 1]
```diff
@@ -11,7 +11,7 @@ PRET
 <reply>
 
 <servercmd>
-REPLY PRET 550 unkown command
+REPLY PRET 550 unknown command
```
Comment (luzpaz) [inline tests/data/test1108]: please review

File: tests/data/test155 [hunk 2]
```diff
@@ -39,7 +39,7 @@ This is not the real page either!
 # This is supposed to be returned when the server gets the second
 # Authorization: NTLM line passed-in from the client
 <data1002>
-HTTP/1.1 200 Type-3 Recevied and all Things are fine swsclose
+HTTP/1.1 200 Type-3 Received and all Things are fine swsclose
```
Comment (luzpaz) [inline tests/data/test155]: please review

File: tests/data/test155 [hunk 3]
```diff
@@ -62,7 +62,7 @@ Content-Length: 34
 Content-Type: text/html; charset=iso-8859-1
 WWW-Authenticate: NTLM TlRMTVNTUAACAAAAAgACADAAAAAGgoEAc51AYVDgyNcAAAAAAAAAAG4AbgAyAAAAQ0MCAAQAQwBDAAEAEgBFAEwASQBTAEEAQgBFAFQASAAEABgAYwBjAC4AaQBjAGUAZABlAHYALgBuAHUAAwAsAGUAbABpAHMAYQBiAGUAdABoAC4AYwBjAC4AaQBjAGUAZABlAHYALgBuAHUAAAAAAA==
 
-HTTP/1.1 200 Type-3 Recevied and all Things are fine swsclose
+HTTP/1.1 200 Type-3 Received and all Things are fine swsclose
```
Comment (luzpaz) [inline tests/data/test155]: same

**Comment by bagder** [comment]
Since the tests all ran fine with this PR, I figure the test changes are fine! :+1:

**Comment by bagder** [comment]
Thanks!

**Pull request closed without merging**